### ISSUE TITLE
Case of enum literals lost in information schema

### ISSUE THREAD

**Issue by knutwannheden** [dolthub/dolt#2391]
As shown by the following example the case of the enum literals is lost in the information schema when creating a table with an enum typed column:
```sql
create table foo(x enum('A','B','C') default 'A')
;
select data_type, column_default
from information_schema.columns
where table_name = 'foo'
and column_name = 'x'
;
```
The latter query returns:
```
+-----------------+--------------+
|data_type        |column_default|
+-----------------+--------------+
|enum('a','b','c')|"A"           |
+-----------------+--------------+
```
Also slightly unusual is that the default value is in double-quotes rather than single quotes.

I could confirm that the constraint is correctly evaluated (i.e. I cannot insert a value `a`, only `A`). But if I were to create a new table based on the output of `SHOW CREATE TABLE` I would get into trouble...

**Comment by knutwannheden** [comment]
Possibly the double quoted default value isn't unusual. I actually didn't double check with what MySQL does here...

**Comment by fulghum** [comment]
Hey @knutwannheden, thanks for opening this issue. I just noticed that we have two issues open for preserving enum case, so to tidy up our backlog, I'm going to close this one in favor of: https://github.com/dolthub/dolt/issues/3991  I'll see if we can prioritize this fix in the next week or so since multiple customers have requested it. 

Hope you are still enjoying Dolt. Please don't hesitate to open more issues or [ping us Discord](https://discord.com/invite/RFwfYpu) if you hit any issues at all with Dolt!

**Comment by fulghum** [comment]
Hey @knutwannheden, just wanted to let you know that the fix for this issue was just released in [Dolt 0.40.26](https://github.com/dolthub/dolt/releases/tag/v0.40.26).

Thanks for taking the time to report this one. Please don't hesitate to cut us more issues if you notice anything else odd!
